js/app.js

```diff
@@ -20,6 +20,13 @@ controller('SponsorsController', function($scope) {
       FacebookLink: 'https://www.facebook.com/pages/%CE%A0%CE%91%CE%93%CE%9A%CE%A1%CE%97%CE%A4%CE%99%CE%91-%CE%A4%CE%A1%CE%91%CE%A0%CE%95%CE%96%CE%91PANCRETA-BANK/139122389454549',
       LinkedInLink: 'https://www.linkedin.com/company/pancretabank/'
     },
+    {
+      Name: 'Webtrails',
+      Image: 'Webtrails_Logo.png',
```

Comment:
Minor typo should be Webtrails_logo.png otherwise the picture is not shown.
![Devstaff - The developers' community gathering - Mozilla Firefox](https://user-images.githubusercontent.com/2233663/72756949-66fbb200-3bd7-11ea-90b8-dd0227598763.png)

Reply:
Should have been fixe by https://github.com/devstaff-crete/devstaff.gr-website-static/pull/96/commits/8729def5741de89d678a60cc15f3d85af1fcc8cb